### DISCUSSION TITLE
SSI E-ID (ID-Karte & Pass)

### DISCUSSION THREAD

**Discussion by meierthomas8** [e-id-admin/open-source-community#15] [General]
Hallo zusammen. Ich befasse mich noch nicht solang mit dem Thema E-ID. Mich würde jedoch folgender Use-Case interessieren: 
Die ID-Karte und der Pass sollen langfristig mit SSI "ersetzt" werden. Wie kann ich als Bürger meine Identität weiterhin ausweisen, wenn z.B. die Infrastruktur (Stromversorgung) zusammenbricht und ich meine elektronischen Geräte nicht mehr laden kann? 

Herzlichen Dank
Thomas

**Comment by grt-dti** [comment]
ich hoffe (ja bete) dafür, dass die nationale E-ID nie ID-Karte und Pass **ersetzen** wird - dafür müsste sich die ganze Welt auf einen neuen Standard von elektronischen Pässen einigen und das letzte Mal (ICAO) hat es über 10 Jahre gedauert und am Ende ein Machtwort der Amerikaner gebraucht (und es spielen immer noch nicht alle Staaten mit). Auch die ID-Karte (also das Plastik-Teil) wird sich wahrscheinlich nie ersetzen (nur ergänzen) lassen, weil sich viel zu viele Notfallprozesse darauf abstützen müssen; da ist die Stromversorgung der Infrastruktur für die Nutzung der E-ID das kleinste aller Problemstellungen - ein Smartphone wird man immer dezentral geladen bekommen (im schlimmsten Fall mit Sonnenkollektoren). Wie das mit einem Blockchainknoten aussieht, steht da auf einem anderen Blatt.... Haha; noch ein Grund gegen eine SSI auf Blockchainbasis.....

**Reply by pascalgottret** [reply]
Die e-ID als Credential wäre mit SSI nichts anderes als ein Datensatz, den du beliebig darstellen lassen kannst. Z.B. mit einem QR Code. Ich gehe ebenfalls davon aus, dass es zumindest nicht langfristig kein Ersatz ist, sondern eine Ergänzung. Dieser QR-Code könnte im physischen Pass erweitert und als kryptographischen Proof verwendet werden.
Zu behaupten, dass dezentrale Blockchainknoten, die im Rahmen eines DID Resolvers für den Issuer Public Key benötigt werden, eine geringere Ausfallsicherheit haben als ein zentrales System sehe ich anders. Deswegen sind sie verteilt und x-fach abgesichert. Wenn ein Knoten ausfällt sind andere ansprechbar. Facebook und Google zeigen ja, dass Ausfälle vorkommen können.

**Reply by grt-dti** [reply]
> ..... Zu behaupten, dass dezentrale Blockchainknoten, die im Rahmen eines DID Resolvers für den Issuer Public Key benötigt werden, eine geringere Ausfallsicherheit haben als ein zentrales System sehe ich anders. Deswegen sind sie verteilt und x-fach abgesichert. Wenn ein Knoten ausfällt sind andere ansprechbar. Facebook und Google zeigen ja, dass Ausfälle vorkommen können.

Das hab ich nicht behauptet  - ich hab nur gesagt, dass die Energieversorgung eines Blockchainknotens eine ganz andere Grössenordnung ist als die Versorgung eines Smartphones mit Energie.
Allerdings ist die zentrale Positionierung einer Blockchain (nicht als einzelne Instanz eines grossen Netzwerks sondern als zentrale kryptographische Hülle einer Datensammlung und technologische Sicherungsinstanz der Prozesstreue) schon ein neuralgisches Element, welches einen zentralen Angriffspunkt darstellen wird. Die Diskussion über die Verlässlichkeit und Integrität einer Blockchain (ganz besonders wenn sich ganze Ökosysteme im Multi-Milliarden Franken Bereich darauf verlassen und demokratische Grundordnungen darauf abstellen) muss erst noch geführt werden.....

**Comment by christianheimann** [comment]
**Die E-ID wird weder ID-Karte noch Pass ersetzen.** Sogar im Gegenteil: Die Ausweisdokumente bilden die Grundlage zur Ausstellung und dem Erhalt einer E-ID. Die E-ID stellt ein digitales Abbild von in der Schweiz genutzten Ausweisdokumenten dar und ist somit als Erweiterung dieser Dokumente zu verstehen.
